openedx/features/caliper_tracking/tests/expected/oppia.exploration.state.changed.json

```diff
@@ -0,0 +1,47 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "action": "JumpedTo",
```

Comment:
Even though we have discussed this, taking a 2nd look, I believe `Modified` is the correct action here.

Reply:
Done in latest commit